src/app/server.settings/server.settings.component.ts

```diff
@@ -189,11 +198,38 @@ export class ServerSettingsComponent implements OnInit, AfterViewInit{
             this.licenseStatus = "Invalid";
             console.log("invalid license");
 
+            if (this.currentLicence.status == this.licenseServerRequestError){
+
+                licenseErrorTitle = "Could Not Connect To License Server"
+                licenseStatusExplaination = "Please Check Your Connection"
+            }
```

Comment:
If it is licenseServerRequestError, I think the code below should not be executed.